.github/workflows/comment-on-pr.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  pull_request:
+
+env:
+  GITHUB_ACCESS_TOKEN: ${{ secrets.DOCKERHUB_USERNAME }}
+  PR_NUMBER: ${{ github.event.number }}
+jobs:
+  comment_on_pr:
+    name: PR comment
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check ENV
+        env:
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+        run: echo $DOCKERHUB_USERNAME
```

Comment:
we should avoid `echo` of env variables unless it is for debugging or other required purposes.

Reply:
Removing that 😅